builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -169,6 +169,20 @@ node() {
       }
       events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', 'pre-build validation completed', context_map, environment_logical_id)
     }
+    stage('Install Plugins') {
+      events.sendStartedEvent('INSTALL_PLUGINS', 'Plugin Installation started', context_map, environment_logical_id)
+      try {
+        send_status_email(config, 'STARTED', "")
```

Comment:
Do we need to send status mail over here ?

Reply:
Removed.